templates/transaction.template

```diff
@@ -12,11 +12,20 @@
       <tr>
 	<td>Siacoin Inputs</td>
 	<td>
-	  <ul>
+	  <table>
+	    <tr>
+	      <td>Output ID</td>
+	      <td>Amount</td>
+	      <td>Address</td>
+	    </tr>
 	    {{range .Tx.SiacoinInputs}}
-	    <li><a href='hash?h={{printf "%x" .ParentID}}'>{{printf "%x" .ParentID}}</a></li>
+	    {{$output := findOutput .ParentID}}
+	    <tr>
+	      <td><a href='hash?h={{printf "%x" .ParentID}}'>{{printf "%x" .ParentID}}</a></td>
+	      <td>{{siacoinString $output.Value}}</td>
```

Comment:
The explorer prints out siacoin amounts in a lot of places. Showing the amount in hastings would be repeat information that probably isn't useful to many people. In the sake of keeping the page clean (as much as this can be said for the current block explorer), I'll leave the output as it currently is for now.